genesis-applier/genesis_applier.go

```diff
@@ -0,0 +1,36 @@
+package genesis_applier
```

Comment:
nit: I think convention for golang package names is single word, no underscores https://go.dev/blog/package-names#:~:text=They%20are%20lower%20case%2C%20with%20no%20under_scores%20or%20mixedCaps

Reply:
interesting, I'll update this along with op_simulator